Python/other/find_Triplets_with_zero_sum.py

```diff
@@ -0,0 +1,88 @@
+'''
+  Author : Mohit Kumar
+  
+  python program to find triplets in a given  array whose sum is zero 
+
+'''
+
+# function to print triplets with 0 sum 
+
+def find_Triplets_with_zero_sum(arr, n): 
```

Comment:
```suggestion
def find_triplets_with_zero_sum(arr,num):
```

Reply:
here n refers to size of input array

Reply:
Then keep it as `length` it would mean more sense